Gemfile

```diff
@@ -4,20 +4,14 @@ source ENV['GEM_SOURCE'] || 'https://rubygems.org'
 
 group :test do
   gem 'coveralls',                 require: false
-  gem 'puppet_metadata', '~> 1.0', require: false
+  gem 'puppet_metadata', '~> 4.0', require: false
   gem 'simplecov-console',         require: false
-  gem 'voxpupuli-test', '~> 5.4',  require: false
+  gem 'voxpupuli-test', '~> 9.0',  require: false
 
-  gem 'puppet-lint-package_ensure-check', '~> 0.2.0',          require: false
-  gem 'puppet-lint-resource_reference_syntax', '~> 1.2.0',     require: false
-  gem 'puppet-lint-strict_indent-check', '~> 2.1.0',           require: false
-  gem 'puppet-lint-unquoted_string-check', '~> 2.2.0',         require: false
-  gem 'puppet-lint-variable_contains_upcase', '~> 1.4.0',      require: false
-
-  gem 'facterdb', '< 2', require: false # 3.0.0 drops support for EL7
-
-  gem 'r10k',    require: false
-  gem 'toml-rb', require: false # puppet/telegraf
+  gem 'librarian-puppet',                             require: false
+  gem 'puppet-lint-package_ensure-check', '~> 0.2.0', require: false
```

Comment:
I asked in IRC/Slack if we should add it to voxpupuli-puppet-lint-plugins, I think that makes sense.

Reply:
Sounds good. I'll remove the direct dep after the next voxpupuli-test release.